### PR TITLE
feat: sanity checker and config subagents

### PR DESCRIPTION
**Pull request by vgorkavenko**

### CODE REVIEW

File: ethereum-operations/src/subagents/oracle-daemon-config/constants.testnet.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    address: ORACLE_DAEMON_CONFIG_ADDRESS,
+    event: "event ConfigValueUnset(string key)",
+    alertId: "ORACLE-DAEMON-CONFIG-VALUE-UNSET",
+    name: "⚠️ Oracle Daemon Config: key was removed",
+    description: (args: any) => args.key,
+    severity: FindingSeverity.High,
+  },
+  {
+    address: ORACLE_DAEMON_CONFIG_ADDRESS,
+    event: "event ConfigValueSet(string key, bytes value)",
```
Comment (madlabman) [inline ethereum-operations/src/subagents/oracle-daemon-config/constants.testnet.ts:26]: Doubles the `ORACLE-DAEMON-CONFIG-VALUE-SET` above

File: ethereum-operations/src/subagents/oracle-daemon-config/constants.testnet.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import { ethers, FindingSeverity } from "forta-agent";
+
+export const ORACLE_DAEMON_CONFIG_ADDRESS =
+  "0xe9cc5bd91543cdc9788454ee5063e2cd76b5206d";
+
+export const ORACLE_DAEMON_CONFIG_EVENTS_OF_NOTICE = [
```
Comment (madlabman) [inline ethereum-operations/src/subagents/oracle-daemon-config/constants.testnet.ts:6]: Could be grabbed from the mainnet.

File: ethereum-operations/src/subagents/oracle-daemon-config/constants.testnet.ts (hunk 1 above)
Comment (vgorkavenko) [reply ethereum-operations/src/subagents/oracle-daemon-config/constants.testnet.ts]: fixed

File: ethereum-operations/src/subagents/oracle-daemon-config/constants.testnet.ts (hunk 2 above)
Comment (vgorkavenko) [reply ethereum-operations/src/subagents/oracle-daemon-config/constants.testnet.ts]: fixed

**Pull request merged**